### PR TITLE
Add server_capabilities API docs

### PR DESCRIPTION
**Pull request by rawlinp**
## What does this PR (Pull Request) do?
Add documentation for the server_capabilities API (depends on merge of https://github.com/apache/trafficcontrol/pull/3966).

- [x] This PR is not related to any Issue 

## Which Traffic Control components are affected by this PR?

- Documentation

## What is the best way to verify this PR?

Build the TC documentation, read the resulting output.

## The following criteria are ALL met by this PR

- [x] Docs only, no tests needed
- [x] This PR includes documentation OR I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR ensures that database migration sequence is correct OR this PR does not include a database migration
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

### CODE REVIEW

File: docs/source/api/server_capabilities.rst [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+		],
+		"response": {
+			"name": "RAM",
+			"lastUpdated": "2019-10-07 22:10:00+00"
+		}
+	}
+
+``DELETE``
+==========
+Deletes a specific tenant.
```
Comment (mhoppa) [inline docs/source/api/server_capabilities.rst]: deletes a specific server capability

Comment (rawlinp) [reply docs/source/api/server_capabilities.rst]: copypasta! thanks

**Comment by rawlinp** [comment]
Sure, I can take a shot, but we should probably have more holistic documentation PR describing how to use all the endpoints together.

**Comment by mhoppa** [comment]
Im fine waiting for that @rawlinp actually might be good because then we can include screen shots of the front end and use of the endpoints together

File: docs/source/api/server_capabilities.rst [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-server_capabilities:
+
+***********************
+``server_capabilities``
+***********************
+
```
Comment (ocket8888) [inline docs/source/api/server_capabilities.rst]: This endpoint was added in version 1.4 - use the `.. versionadded::` directive to indicate that.

**Pull request merged**